cwe_api/Dockerfile

```
@@ -3,10 +3,26 @@ FROM python:3.12-slim
+    && apt-get install -y --no-install-recommends \
+       wget \
+       unzip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    || { echo "Error: Fallo en la instalación de paquetes"; exit 1; }
 
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir torchvision==0.17.0+cpu torchaudio==2.2.0+cpu --index-url https://download.pytorch.org/whl/cpu \
+    && pip install --no-cache-dir -r requirements.txt
+
+RUN if [ ! -d modelo_cwe ]; then \
```

Hadolint

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)